homework/AdditionalTaskCSharpOOP/AdditionalTasks.cs

```diff
@@ -0,0 +1,34 @@
+﻿
+namespace AdditionalTaskCSharpOOP
+{
+    public class AdditionalTasks
+    {
+        //Task_1
+        public double CurrencyConverter(double byn, double usd)
```

Comment:
suggestion: not usd, but exchangeRate. Also not CurrencyConverter, but ConvertCurrency :) Remember naming rules :)